recipes/chartlets/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "chartlets" %}
+{% set version = "0.0.29" %}
+{% set python_min = "3.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/bcdev/chartlets/archive/{{ version }}.tar.gz
+  sha256: d3fb3d5c68d3af5fd93ecab5cc5a033ef4bec88a5b0d9c8630cdc0827021f09c
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - setuptools >=61.2.0
+    - pip
+  run:
+    - python {{ python_min }}
+    - altair
```

Comment:
`altair` is not required.
Please check `environment.yml`, only include required packages.